src/pages/overview/pdf-electronic-seal-api/quickstarts/java/index.md

```diff
@@ -54,16 +54,17 @@ To complete this guide, you will need:
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.adobe.documentservices</groupId>
-  <artifactId>pdfservices-sdk-electronicseal-guide</artifactId>
-  <version>1</version>
+  <artifactId>pdfservices-sdk-samples</artifactId>

```

Comment:
this name needs to be changed across all quickstart pages
Earlier name was indicative of the mega operation being performed like "pdfservices-sdk-electronicseal-guide".

Reply:
Reverted.